scripts/generator/graphql_generator/csharp/SDK/CartLineItems.cs.erb

```diff
@@ -376,10 +362,39 @@ namespace <%= namespace %>.SDK {
             return Delete(variantId);
         }
 
-        private void OnCartLineItemChange(CartLineItem modifiedLineItem) {
-            if (!LineItemsNew.Contains(modifiedLineItem) && !LineItemsModified.Contains(modifiedLineItem)) {
-                LineItemsModified.Add(modifiedLineItem);
+        public void UpdateLineItemsFromCheckoutLineItems(List<CheckoutLineItem> checkoutLineItems) {
```

Comment:
I was thinking before we merge the `beta` branch on `master` to do a full review of documentation.